### PR TITLE
[subscribe-button] Add subscribe button to Winging It Posts

### PR DESCRIPTION
**Pull request by dvdherron**
## Steps to test/reproduce

- navigate to `/2023/07/20/winging-it-01/`

_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me

image

_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Comment by dvdherron** [comment]
[Add Subscribe Button to OddSite Winging It Posts](https://trello.com/c/Ati0nwQf/225-add-subscribe-button-to-oddsite-winging-it-posts)

**Comment by dvdherron** [comment]
@jgerigmeyer @stacyk I got this to work by adding an argument to the ` embed.figure` macro as suggested but I'm not sure I'm calling everything right. 

And it doesn't seem like the [code that YouTube provides](https://developers.google.com/youtube/youtube_subscribe_button) can be edited (in case we want to add options for the switching the layout of the button from `default` or `full` depending on where we add this button). Can either of you see a way to make that work in our macros?

**Comment by jgerigmeyer** [comment]
> @jgerigmeyer @stacyk I got this to work by adding an argument to the ` embed.figure` macro as suggested but I'm not sure I'm calling everything right.
> 
> And it doesn't seem like the [code that YouTube provides](https://developers.google.com/youtube/youtube_subscribe_button) can be edited (in case we want to add options for switching the layout of the button from `default` or `full` depending on where we add this button). Can either of you see a way to make that work in our macros?

@dvdherron I pushed some changes in https://github.com/oddbird/oddleventy/commit/7ce86372c4bc9f054ae312fedc0948b00c064718 which allow calling the macro with either `default` or `full` layouts.

**Comment by dvdherron** [comment]
@jgerigmeyer I had to move the macro around to get this to work as part of the `main_action` macro in `intro.njk`

@stacyk @mirisuzanne 
Any idea why the YouTube button text is distorted when the intro is rotated?

bild

bild

TODO: If we end up leaving this button here we probably want to make the `Subscribe to Channel` text bold or styled in a different way too.

**Comment by stacyk** [comment]
>
> @stacyk @mirisuzanne Any idea why the YouTube button text is distorted when the intro is rotated?
> 

> TODO: If we end up leaving this button here we probably want to make the `Subscribe to Channel` text bold or styled in a different way too.

hmm, it looks smooth on a Mac in Safari, Chrome and FF. I'm signed in on Chrome which is why there is a checkmark instead of the play button. 

I tried a few CSS tricks but it is hard for me to test since it looks smooth to begin with.


I agree that we should style the "Subscribe to Channel »" as that currently looks like a link with the Chevrons.

firefox
chrome
safari

**Comment by stacyk** [comment]
I tried to play with a few ideas in sauce labs too @dvdherron. Was this jagged screenshot taken when using Firefox? I wasn't able to find a quick solution. Maybe @mirisuzanne has some ideas?

**Comment by dvdherron** [comment]
This is now a main-action link on all Winging It posts and the main Winging It page:

image
Main page

image

post

image
post


I think this is ready for review for all who want to take a look @stacyk @mirisuzanne @SondraE @jgerigmeyer

File: content/_includes/embed.macros.njk [hunk 1]
```diff
@@ -95,6 +95,9 @@ params:
   class:
     type: string
     default: "''"
+  attrs:
+    type: object
+    default: '{}'
```
Comment (stacyk) [inline content/_includes/embed.macros.njk:100]: @dvdherron is this still useful since we moved the subscribe link to utilities?

**Review by stacyk — APPROVED** [review]
@dvdherron looks good! Left one question for you, but otherwise this is good to go

**Review by jgerigmeyer — APPROVED** [review]

**Review by mirisuzanne — APPROVED** [review]
looks great

**Pull request merged**